moveit_ros/moveit_servo/include/moveit_servo/low_pass_filter.h

```diff
@@ -63,7 +63,7 @@ class LowPassFilter
   double previous_measurements_[FILTER_LENGTH];
   double previous_filtered_measurement_;
   // Scale and feedback term are calculated from supplied filter coefficient
-  const double scale_term_;
-  const double feedback_term_;
+  double scale_term_;
```

Comment:
Unrelated to your changes, but ...

Would you mind deleting the TODO at Line 49? I don't see it happening anytime soon, because this filter is very lightweight and well-suited to the application.

Reply:
Sure. I'll do that when I add the test